.eslintrc.json

```diff
@@ -1,11 +1,3 @@
 {
-  "extends": ["@bjerk/eslint-config", "plugin:@next/next/recommended"],
-  "overrides": [
-    {
-      "files": "src/pages/**/*.tsx",
-      "rules": {
-        "import/no-default-export": "off"
-      }
-    }
-  ]
+  "extends": "next/core-web-vitals"
 }
```

Comment:
We should open a PR where we extend bjerk's eslint config in addition to the next eslint config